hardware_interface/include/joint_limits_interface/joint_limits_interface.hpp

```diff
@@ -17,7 +17,7 @@
 #ifndef JOINT_LIMITS_INTERFACE__JOINT_LIMITS_INTERFACE_HPP_
 #define JOINT_LIMITS_INTERFACE__JOINT_LIMITS_INTERFACE_HPP_
 
-#include <hardware_interface/joint_handle.hpp>
+#include <hardware_interface/handle.hpp>
```

Comment:
Use standardized interface types

```suggestion
#include <hardware_interface/handle.hpp>
#include <hardware_interface/types/hardware_interface_type_values.hpp>
```